packit/service/web_hook.py

```diff
@@ -58,6 +58,30 @@ def github_release():
         logger.debug(f"/webhooks/github/release received ping event: {msg['hook']}")
         return "Pong!"
 
+    return _give_event_to_steve(msg)
+
+
+@app.route("/webhooks/github/pull_request")
```

Comment:
I actually fear that we'll have to have only a single webhook endpoint for github. I just realized that github app interface doesn't allow endpoint per webhook type (just a headsup, we can refactor this in the future.